bin/generate-changelog

```diff
@@ -59,16 +60,29 @@ function getChangelog(SymfonyStyle $ui, string $repository, $range): string
             $author = $commit['commit']['author']['name'];
         }
 
+        $authors[] = $author;
+
         // replace #issue-number with sulu/sulu#issue-number so the changelog can also be used for sulu/skeleton
         $message = \str_replace('#', $repository . '#', $message);
 
         $lines[] = \sprintf(' - %s (%s)', $message, $author);
     }
 
     $lines = \array_reverse($lines);
+    $authors = \array_unique($authors);
+    \sort($authors);
 
     $changelog .= implode(PHP_EOL, $lines);
 
+    if ('n' !== \strtolower($ui->ask(\sprintf('Show stats (merged pull requests, contributers)?'), 'n'))) {
+        $changelog .= PHP_EOL;
+        $changelog .= PHP_EOL . 'Merged Pull Requests: ' . count($lines);
+        $changelog .= PHP_EOL;
+        $changelog .= PHP_EOL . 'Contributers: ' . count($authors);
+        $changelog .= PHP_EOL;
+        $changelog .= '  ' . implode(PHP_EOL . '   ', $authors);
```

Comment:
are the three spaces in the `implode` function intentional?

Reply:
was but did remove it as copy paste of the list was not great then #4